src/compiler/factoryPublic.ts

```diff
@@ -3448,7 +3448,7 @@ namespace ts {
      * Gets a custom text range to use when emitting source maps.
      */
     export function getSourceMapRange(node: Node): SourceMapRange {
-        const emitNode = node.emitNode;
+        const emitNode = node && node.emitNode;

```

Comment:
If we're getting an error here its because we are somehow being passed a `node` that is `undefined` however we expect a defined node.

Reply:
_Probably_ a `symbol.valueDeclaration` via some means, since that's a known hole in our current type definitions.